code/datums/supply_packs/explosives.dm

```diff
@@ -67,6 +67,21 @@
 	containername = "\improper plastic explosives crate (WARNING)"
 	group = "Explosives"
 
+/datum/supply_packs/brute_rockets
+	name = "BRUTE rocket crate (x6)"
```

Comment:
something like `"M6H Breaching Rocket Crate (x6)"` may be more clear to req players who won't necessarily know wtf a BRUTE is.